core/src/main/java/com/linecorp/armeria/server/HttpServerHandler.java

```diff
@@ -400,6 +400,12 @@ private void handleRequest(ChannelHandlerContext ctx, DecodedHttpRequest req) th
                 res = serveInServiceEventLoop(req, serviceCfg, service, reqCtx, serviceEventLoop);
             }
         }
+        res = res.recover(cause -> {
+            reqCtx.logBuilder().responseCause(cause);
+            // Recover the failed response with the error handler.
+            return serviceCfg.errorHandler().onServiceException(reqCtx, cause);
```

Comment:
nit: We can remove the `serviceCfg` parameter in `serve0`.

Reply:
Fixed. 😉